apps/desktop/src/components/views/terminal/terminal-tab-button.tsx

```
@@ -0,0 +1,100 @@
+          className="w-20 rounded border border-border bg-background px-1 py-0 text-sm focus:outline-none focus:ring-1 focus:ring-primary"
+        />
+      ) : (
+        <span className="max-w-24 truncate text-sm">{tab.name}</span>
+      )}
+      <button
+        onClick={(e) => {
+          e.stopPropagation();
+          onClose();
+        }}
+        className="ml-1 rounded p-0.5 opacity-0 hover:bg-accent group-hover:opacity-100"
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)